.gitignore

```diff
@@ -1 +1,2 @@
 /target
+Cargo.lock
```

Comment:
It's common to include Cargo.lock files for binaries such as this, but exclude it for libraries. 

https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries